wp-content/themes/core/blocks/core/lists/style.pcss

```diff
@@ -6,7 +6,7 @@
  * Ordered List
  * ---------------------------------------------------------------------- */
 
-:where(.wp-site-blocks) ol,
+.wp-site-blocks ol:not([class]),
```

Comment:
Talk me through this selector & change:

It looks like we're targeting an `ol` inside `.wp-site-blocks` that does not have a class attribute, correct? Then, on the next line, we're also targeting any `ol` _with_ a class of `wp-block-list`? 

I'm guessing this is needed for differences between the editor and the FE, but I'm wondering if we can can reduce/simplify it more?

Reply:
The styles in this file are supposed to be specific to the List block. The problem is that the list block (much like the paragraph block) doesn't get a class on the FE, which makes it difficult to style and we've been having these styles bleed out into other blocks. Fortunately, other blocks that use lists on the FE _do_ get classes (Post Template block / Query block was a big pain point, Categories block, etc.). I've found the only list elements that don't get classes come from the list block (ironic 🙄).

In the editor, the List block _does_ get a class of `wp-block-list`, so that's easy to target and we don't need to prefix, because the `editor-styles-wrapper` prefix happens automatically. 

The FE selector will also get prefixed but it won't do anything because I believe `editor-styles-wrapper` and `wp-site-blocks` are the same element in the editor.

Reply:
Thanks for clarifying. Merge away!